env.template

```diff
@@ -1 +1,2 @@
 export FLASK_ENV=development
+export YELP_API_KEY=<insert>
```

Comment:
same thing here, also maybe just put `API_KEY` instead since we don't have the `<>` convention anywhere else?